.prettierignore

```diff
@@ -9,7 +9,13 @@ CHANGELOG.md
 .husky/_/
 .changeset/*.md
 
-# temporarly ignore follow files because prettier-ignore comments don't work in mdx2
+# temporarily ignore follow files because prettier-ignore comments don't work in mdx2
 # see https://github.com/prettier/prettier/pull/12208
-website/docs/advanced/how-does-it-work.mdx
-website/docs/getting-started/development-workflow.mdx
+website/src/pages/docs/advanced/generated-files-colocation.mdx
```

Comment:
should we ignore all `.mdx`?

Reply:
if not all we can also just put prettier ignore in specific files

Reply:
Prettier ignore comments currently doesn’t work in mdx2

Reply:
Very sad, there is pr with fix from February but they didn’t merged it 😂